README.md

```diff
@@ -11,7 +11,7 @@ forge install brockelmore/forge-std
 ## Contracts
 ### stdError
 
-This is a helper contract for errors and reverts in solidity. In `forge`, this contract is particularly helpful for the `expectRevert` cheatcode, as it provides all compiler builtin errors.
```

Comment:
I was gonna capitalize Solidity but then thought maybe its better just to delete it...